ci-operator/config/openshift/installer/openshift-installer-master.yaml

```diff
@@ -422,8 +420,7 @@ tests:
     - ref: ipi-conf-aws-byo-ipv4-pool-public
     - ref: aws-provision-iam-profile
     - ref: ipi-conf-aws-byo-iam-profile
-    - ref: ipi-conf-aws-user-min-permissions
-    - ref: aws-provision-iam-user
+    - chain: aws-provision-iam-user-minimal-permission
```

Comment:
`- ref: ipi-conf-aws-user-min-permissions` and `- ref: aws-provision-iam-user` were merged into `- chain: aws-provision-iam-user-minimal-permission`

Reply:
Nice